bots/twitch/phantombot/README.md

```diff
@@ -1,8 +1,6 @@
 # PhantomBot
 ### Their [Site](https://phantombot.github.io/PhantomBot/)
-PhantomBot is an actively developed open source interactive Twitch bot
-with a vibrant community that provides entertainment and moderation for your channel,
-allowing you to focus on what matters the most to you - your game and your viewers.
+PhantomBot is an actively developed open source interactive Twitch bot with a vibrant community that provides entertainment and moderation for your channel, allowing you to focus on what matters the most to you - your game and your viewers.
```

Comment:
This is broken like this to be readable in a text editor instead of one long line.

Markdown treats these the same, while one is much less readable.